bin/v-add-firewall-chain

```diff
@@ -47,8 +47,8 @@ fi
 #                    Verifications                         #
 #----------------------------------------------------------#
 
-check_args '1' "$#" 'CHAIN [PORT] [PROTOCOL]'
-#is_format_valid 'chain' 'port_ext' 'protocol'
+check_args '1' "$#" 'CHAIN [PORT] [PROTOCOL] [FAMILY] [IPTABLES] [LOCKINGOPT]'
```

Comment:
Please make sure to add validation for this ...

Reply:
For optional parameters too?
I will show, how it is implemented with other checks. But you are right, the [IPTABLES] parameter is very dangerous. Theoretically you can define instead of iptables each other binary or script here.

Reply:
These parameters are from standard examples and scripts, proposed by fail2ban

Reply:
We need at least prevent users can't be using  '  as it can be abused...

format_no_quotes should fix it ...

Reply:
No, if I implement it, then not a half solution. I make it completelly. [IPTABLES] is really very dangerous and I will check it double. [FAMILY] can be only 'inet4' or 'inet6', therefore it is very simple to check it. I rename then the variable to a better and unique one, since "family" is very common. [LOCKINGOPT] is deactivated now, since I used it wrong with quotes. But I will activate it in future and check it against '-w', since it can have actually only this option.